src/util/numeric.h

```diff
@@ -144,36 +125,17 @@ inline bool isInArea(v3s16 p, v3s16 d)
 	);
 }
 
-#define rangelim(d, min, max) ((d) < (min) ? (min) : ((d)>(max)?(max):(d)))
-#define myfloor(x) ((x) > 0.0 ? (int)(x) : (int)(x) - 1)
-
 inline v3s16 arealim(v3s16 p, s16 d)
 {
-	if(p.X < 0)
```

Comment:
It helps if you put comments below the changed code so I don't have to search for it in the "Files changed" tab.  Also, there's a clear difference between that example and this: The following line isn't deceptively indented here.  But if that's what you want it can be done...